shared/chat/inbox/index.native.tsx

```diff
@@ -260,21 +260,17 @@ class Inbox extends React.PureComponent<T.Props, State> {
     const floatingDivider = this.state.showFloating &&
       !this.props.isSearching &&
       this.props.allowShowFloatingButton && <BigTeamsDivider toggle={this.props.toggleSmallTeamsExpanded} />
-    const HeadComponent = <ChatInboxHeader context="inbox-header" />
     return (
       <Kb.ErrorBoundary>
         <Kb.Box style={styles.container}>
-          {!!this.props.isLoading && (
-            <Kb.Box style={styles.loadingContainer}>
-              <Kb.LoadingLine />
-            </Kb.Box>
-          )}
+          <LoadingLine isLoading={this.props.isLoading} />
           {this.props.isSearching ? (
             <Kb.Box2 direction="vertical" fullWidth={true}>
               <InboxSearch header={HeadComponent} />
             </Kb.Box2>
           ) : (
             <Kb.NativeFlatList
+              overScrollMode="never"
```

Comment:
else android has this behavior where if you hit the end the components 'smear' and stretch out which feels very weird